resources/eventing/charts/nats/templates/service.yaml

```diff
@@ -8,14 +8,13 @@ metadata:
     {{- include "nats.labels" . | nindent 4 }}
   {{- if .Values.serviceAnnotations}}
   annotations:
-  {{- range $key, $value := .Values.serviceAnnotations }}
-    {{ $key }}: {{ $value | quote }}
-  {{- end }}
+    {{- toYaml .Values.serviceAnnotations | nindent 4 }}
```

Comment:
Please verify if this will also generate the same set of annotations as we have right now.

Reply:
it is generating the same output